frontend/src/component/changeRequest/ChangeRequest/StrategyTooltipLink/StrategyTooltipLink.tsx

```diff
@@ -73,37 +74,47 @@ export const StrategyTooltipLink: FC<IStrategyTooltipLinkProps> = ({
     change,
     previousTitle,
     children,
-}) => (
-    <StyledContainer>
-        <GetFeatureStrategyIcon strategyName={change.payload.name} />
-        <Truncated>
-            <ConditionallyRender
-                condition={Boolean(
-                    (previousTitle && previousTitle !== change.payload.title) ||
-                        true
-                )}
-                show={
-                    <Truncated>
-                        <Typography component="s" color="text.secondary">
-                            {previousTitle}
-                        </Typography>{' '}
-                    </Truncated>
-                }
-            />
+}) => {
+    const { strategies } = useStrategies();
+    const titleHasChanged = Boolean(
+        (previousTitle && previousTitle !== change.payload.title) ||
+            (!previousTitle && change.payload.title)
+    );
+    const previousTitleOrDefault = () =>
```

Comment:
I don't think this has to be a function

Reply:
Doesn't have to, but that way we don't evaluate it when we don't have to 💁🏼 I'm happy to make it a variable though